x-pack/plugins/beats_management/public/components/beat/activity.tsx

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+export const BeatActivityView = (props: any) => <div>Beat Activity View</div>;
```

Comment:
this should be a page, note the pattern in the pages of page index being a parent page, with then activity, detail, and tags being separate files in that folder